manifests/ca.pp

```diff
@@ -124,20 +125,22 @@
     mode   => '0750'
   })
 
-  exec { "copy easy-rsa to openvpn config folder ${name}":
-    command => "/bin/cp -r ${openvpn::params::easyrsa_source} ${etc_directory}/openvpn/${name}/easy-rsa",
+    exec { "copy easy-rsa to openvpn config folder ${name}":
+    command => "/bin/cp -rL ${openvpn::params::easyrsa_source} ${etc_directory}/openvpn/${name}/easy-rsa",
     creates => "${etc_directory}/openvpn/${name}/easy-rsa",
     require => File["${etc_directory}/openvpn/${name}"],
   }
 
-  file { [
-    "${etc_directory}/openvpn/${name}/easy-rsa/clean-all",
-    "${etc_directory}/openvpn/${name}/easy-rsa/build-dh",
-    "${etc_directory}/openvpn/${name}/easy-rsa/pkitool",
-  ]:
-    ensure  => file,
-    mode    => '0550',
-    require => Exec["copy easy-rsa to openvpn config folder ${name}"],
+  if $openvpn::params::easyrsa_ver == '2.0' {
```

Comment:
can we somehow determine the actual rsa version from the system?  One idea would be to parse `easyrsa --help` which prints `Easy-RSA 3` at least on my arch system

Reply:
It might be possible to do this, but it would slow down puppet runs to run exec each time. Setting this via the params based on OS and version should be good enough, it's unlikely to change often.